digitalocean/resource_digitalocean_app_test.go

```diff
@@ -633,3 +664,57 @@ resource "digitalocean_app" "foobar" {
     }
   }
 }`
+
+var testAccCheckDigitalOceanAppConfig_addJob = `
```

Comment:
A job is expected to complete as it is run as part of the deploy process. (Think database migrations.) The `sample-sleeper` program runs indefinitely. So I changed this test up to use something else.